src/applications/vaos/appointment-list/components/RequestedStatusAlert.jsx

```diff
@@ -59,17 +59,20 @@ export default function RequestedStatusAlert({ appointment, facility }) {
             available appointment.
             <br />
             <div className=" vads-u-margin-top--1">
-              <Link to="/" onClick={handleViewClick}>
-                View your appointments
-              </Link>
+              <va-link
+                href="/"
+                onClick={handleViewClick}
+                text="View your appointments"
+                data-testid="view-appointments-link"
+              />
             </div>
             <div className=" vads-u-margin-top--1">
-              <Link
-                to="/new-appointment"
+              <va-link
+                href="/new-appointment"
```

Comment:
This link is going to the wrong path. "/new-appointment" was enrolled in a special routes file. I don't know how it works but @vbahinwillit may have the background on it.

Reply:
@jenniemc I also need to look at why tests were passing, they should not have been. I may need to adjust those.